pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -67,6 +68,8 @@ public class Driver implements java.sql.Driver {
   private static final Logger LOGGER = Logger.getLogger("org.postgresql.Driver");
   private static final SharedTimer SHARED_TIMER = new SharedTimer();
 
+  private static final LazyCleaner CLEANER = new LazyCleaner(10, "Cleaner");
```

Comment:
Thread timeout is in `milliseconds`, and I think we should use longer defaults.

An alternative option is to use `java.time.Duration` instead of `long` for the constructor parameter, so we don't confuse seconds with millis.